tests/ast/test_ast_driver.py

```diff
@@ -94,6 +94,11 @@ def load_test_cases():
     Returns: a list of test cases.
     """
     test_files = DATA_DIR.glob("*.test")
+    if sys.version_info[1] < 9:
```

Comment:
Should probably check that `[0]` == 3. Your grandchildren who will have to maintain this code will thank you ;).